core/controllers/base.py

```diff
@@ -90,7 +90,7 @@ def get(self):
         auth_services.destroy_auth_session(self.response)
 
 
-class UserFacingExceptions(python_utils.OBJECT):
+class UserFacingExceptions(object):
```

Comment:
Actually, while you are removing this, when object is inside class definition, you can do this:
```suggestion
class UserFacingExceptions:
```

Reply:
ditto in all the other places

Reply:
will do that. Being new to GitHub I was not able to resolve merge conflict between my local and remote. So I'm working with a new branch. Sorry for inconvience. You can close this PR.

Reply:
On accepting current changes the file was getting overwritten with stashed and current at the same time.